gui/qt/exception_window.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+#
+# Electrum - lightweight Bitcoin client
```

Comment:
I copied this from `qrwindow.py`. Not sure about the copyright notice that was there. Should it include your name or my name?